Export ORANGE input from JSON and add full support for transforms (#976)
celeritas-project/celeritas: doc/usage.rst

.. table:: Geant4 UI commands defined by ``celer-g4``.

 ================== ==================================================
 Command            Description
 ================== ==================================================
 geometryFile       Filename of the GDML detector geometry
 eventFile          Filename of the event input read by HepMC3
 rootBufferSize     Buffer size of output root file [bytes]
 writeSDHits        Write a ROOT output file with hits from the SDs
 stepDiagnostic     Collect the distribution of steps per Geant4 track

>  stepDiagnosticBins Collect the distribution of steps per Geant4 track

Reviewer: ```suggestion
 stepDiagnosticBins Number of bins for the Geant4 step diagnostic
```
